extractorapp/src/deb/resources/etc/georchestra/extractorapp/extractorapp.properties

```diff
@@ -0,0 +1,34 @@
+
+host=georchestra.mydomain.org
+instance=georchestra
+language=en
+
+# The base url of the extractorapp as accessible from outside the intranet
+servletUrl=http://georchestra.mydomain.org/extractorapp
+
+# Hostname of the geoserver that is secured for geOrchestra.
+secureHost=georchestra.mydomain.org
+
+# Email properties
+smtpHost=localhost
+smtpPort=25
+replyTo=psc@georchestra.org
+from=psc@georchestra.org
+emailHtml=false
+
+privileged_admin_name=geoserver_privileged_user
+privileged_admin_pass=gerlsSnFd6SmM
+
+dlformjdbcurl=jdbc:postgresql://localhost:5432/georchestra?user=www-data&password=www-data
```

Comment:
misses a comment here stating that it is optional (required if and only if dlform module is deployed)